packages/snapshot-listener/src/utils/fetchSnapshot.ts

```diff
@@ -0,0 +1,64 @@
+import fetch from "node-fetch";
+import dotenv from "dotenv";
+import { factory, formatFilename } from 'common-common/src/logging';
+
+dotenv.config();
+
+const log = factory.getLogger(formatFilename(__filename));
+
+export default async function fetchNewSnapshotProposal(id: string) {
+  try {
+    const environment = process.env.NODE_ENV || "development";
+
+    if (environment === "development") {
```

Comment:
Should we remove this?

Reply:
I think this is still useful for dev environment, otherwise the developer will get an error when it tries to call snapshot with an invalid / fake ID